update marina doc + add ionio-example (#18)
vulpemventures/docs.vulpem.com: docs/marina/introduction.md

Marina uses an embedded HD wallet and encrypts it with AES-256 the mnemonic seed with a password of choice and we always ask the user to unlock (ie. decrypt) with the password everytime we need to access the private keys to sign a transaction or to show the mnemonic.

> The wallet is divided into accounts. Each account is a one of the sub-hierarchies of the HD wallet. An account is defined by a unique derivation path and an account name. By default, Marina creates _3 "main" accounts_:

Reviewer: ```suggestion
The wallet is organized by accounts. Each account is a different path of the wallet's HD tree and is assigned with a unique name. By default, Marina creates _3 "main" accounts_:
```

 * "mainAccount" (m/84'/1776'/0') enabled on Liquid mainnet only.
 * "mainAccountTest" (m/84'/1'/0') enabled on Liquid testnet and regtest networks.
 * "mainAccountLegacy" (m/84'/0'/0') enabled on all networks.
All the main accounts are confidential P2WPKH (native segwit).

:::tip
The "mainAccountLegacy" is not used by Marina extension to receive coins. It is only used to restore and spend the balance of legacy accounts created with the old version of Marina. 
:::

> Marina lets user to create additional custom accounts. In order to be restorable, all accounts (except main ones) derivation path are deterministic and depends on the __account name__. Marina is using [SLIP13](https://github.com/satoshilabs/slips/blob/master/slip-0013.md) to derive the account path from the account name.

Reviewer: ```suggestion
Marina allows the user to create additional custom accounts and uses [SLIP13](https://github.com/satoshilabs/slips/blob/master/slip-0013.md) to generate the derivation path directly from an account name.
```
